software/scripts/gui.py

```diff
@@ -103,18 +103,21 @@
     defaultFile = args.defaultFile,
 )
 
+# Dump the yaml file
+camLinkDev.saveAddressMap( "CamlinkDev.yaml" )
+
 #################################################################
 
 # # Dump the address map
-# pr.generateAddressMap(cl,'addressMapDummp.txt')
+# pr.generateAddressMap(camLinkDev,'addressMapDummp.txt')
```

Comment:
Can you change `generateAddressMap` to `saveAddressMap`?

Reply:
Did a cleanup replacing stale comment w/ call to saveAddressMap and filename to addressMapDump.txt